primitive-types/impls/serde/Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
 name = "impl-serde"
-version = "0.2.1"
+version = "0.2.2"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "Apache-2.0/MIT"
 homepage = "https://github.com/paritytech/parity-common"
 description = "Serde serialization support for uint and fixed hash."
 
 [dependencies]
-serde = "1.0"
+serde = "1.0.101"
```

Comment:
let's not be too restrictive and revert this change

Reply:
I see your point, but I also wonder why someone would want to use an older patch release of `serde`.  `1.0.101` also matches any higher minor or patch version.

Reply:
Serde doesn't follow semver though and makes patch releases for new features as well, so this doesn't state which features of serde we rely on. On the other hand, I don't see why anyone would not want the latest version.

Reply:
We don't have a well defined policy wrt older rustc versions other than: "compiles on stable at the time of writing (the code)".

Personally I don't mind being explicit about the version here.